pyproject.toml

```diff
@@ -71,7 +72,7 @@ smqtk-nn-index-tool = "smqtk_iqr.utils.nn_index_tool:cli_group"
 [tool.pytest.ini_options]
 addopts = [
     "-lv",                      # Show local in trace-backs.
-    "--doctest-modules",        # Increased verbosity.
+    "--xdoctest-modules",       # Increased verbosity.
```

Comment:
The comment here confused me. Should the comment be "enable running doctests instead"? This doesn't really increase verbosity.

Reply:
Probably a lot of copy paste from back when the packages were being overhauled to use poetry in the first place. I see that comment in the other repos as well.